### ISSUE TITLE
Configuration Object Export

### ISSUE THREAD

**Issue by pierogipowered** [hire-vladimir/SA-cim_validator#17]
Health Check: One or more apps ("SA-cim_vladiator-master") that had previously been imported are not exporting configurations globally to system. Configuration objects not exported to system will be unavailable in Enterprise Security.

https://docs.splunk.com/Documentation/Splunk/8.0.0/Admin/Apparchitectureandobjectownership

**Comment by lakshman237** [comment]
Yes true @pierogipowered . In my instance, I had created metadata/local.meta and provided export=system to address the above, while this gets addressed in the default.meta in future releases by @hire-vladimir
